.github/workflows/nightly-release.yml

```diff
@@ -7,7 +7,7 @@ on:
 
 permissions:
   issues: read
-  pull-requests: read
+  pull-requests: write
```

Comment:
i think this one is ok to be `read`. this workflow needs `contents` scope to be `write` i think because there's a step to create and push a new tag to the repository. https://docs.github.com/en/actions/using-jobs/assigning-permissions-to-jobs

Reply:
fixed.. added job specific permission for contents.